Sources/SwifterSwift/Foundation/MeasurementExtensions.swift

```diff
@@ -49,4 +49,46 @@ public extension Measurement where UnitType == UnitAngle {
     }
 }
 
+public extension Measurement {
+    /// SwifterSwift: Increase `Measurement` by `Measurement`
+    /// - Parameters:
+    ///   - lhs: `Measurement` to increase
+    ///   - rhs: another `Measurement`
```

Comment:
Steal comments from Apple's documentation, it's prettier ;)
```suggestion
    /// SwifterSwift: Adds two values and stores the result in the left-hand-side variable.
    /// - Parameters:
    ///   - lhs: The first value to add.
    ///   - rhs: The second value to add.
```